devicetypes/smartthings/zigbee-smoke-sensor.src/zigbee-smoke-sensor.groovy

```diff
@@ -141,7 +142,7 @@ def ping() {
 
 def configure() {
 	log.debug "configure"
-	sendEvent(name: "checkInterval", value:10 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	sendEvent(name: "checkInterval", value:20 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
Why is this change being made? The maxReportTime for IAS Zone is 180 seconds so it shouldn't be necessary to increase the checkInterval from 12 minutes to 22 minutes

Reply:
the IAS Zone of heiman smoke sensor didn't report event, and battery event interval is 1200 seconds, so we changed the healthcheck interval to 20*60+2*60